src/Components/QuestionListReversedStrings/utils.ts

```diff
@@ -0,0 +1,33 @@
+import { Question } from "../../Apis/types";
+
+interface Props {
+    questions: Question[];
+}
+
+const reverseOddWordsInSentence = (string: string) => {
+    let newString = string.split(" ");
+    return newString
+        .map((word) => {
+            if (word.length % 2 !== 0) {
+                word = word.split("").reverse().join("");
+            }
+            return word;
+        })
+        .join(" ");
+};
+
+export const reverseStringsOddLength = ({ questions }: Props) => {
+    let reverseStringQuestionsOdd = questions;
```

Comment:
I was under the assumption that if 
`const a;`
`const b = a;` 
`b` is a pointer to `a`, but if we manipulate` b` in some fashion, JS makes a new reference to `b` with the updated data.

so if its `b` is a reference to `a`, in order to make this functional would I have to do a map or something that returns an array or would i use the `new` keyword 

so it would be `const b = new a`?